### PR TITLE
Pom cleanup

### PR DESCRIPTION
**Pull request by robertkrantz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -415,13 +415,6 @@
     <build>
         <pluginManagement>
             <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-ejb-plugin</artifactId>
-                    <configuration>
-                        <ejbVersion>3.1</ejbVersion>
-                    </configuration>
-                </plugin>
```
Comment (psjobergprivat) [inline pom.xml:424]: Interesting. Do you know if ejb-jar.xml is used? If it is we should probably update it.

File: pom.xml [hunk 2]
```diff
@@ -451,10 +451,27 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <excludes>
-                        <exclude>**/*ITest.java</exclude>
+                        <exclude>**/*IntegrationTest.java</exclude>
```
Comment (psjobergprivat) [inline pom.xml]: Maybe it's possible to have the value as a property, so the include/exclude cannot diverge

**Review by psjobergprivat — APPROVED** [review]
Spontaneous cleaning definitely has a value!

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Pull request merged**